src/Lean/Server/FileWorker/WidgetRequests.lean

```diff
@@ -53,18 +53,25 @@ def makePopup : WithRpcRef InfoWithCtx → RequestM (RequestTask InfoPopup)
         | none => pure none
       let exprExplicit? ← match i.info with
         | Elab.Info.ofTermInfo ti =>
-          let ti ← ppExprTagged ti.expr (explicit := true)
-          -- remove top-level expression highlight
-          pure <| some <| match ti with
-            | .tag _ tt => tt
-            | tt => tt
+          pure <| some <| ← ppExprTaggedWithoutTopLevelHighlight ti (explicit := true)
+        | Elab.Info.ofOmissionInfo { toTermInfo := ti } =>
+          -- Omitted terms are simply to be expanded, not printed explicitly.
+          pure <| some <| ← ppExprTaggedWithoutTopLevelHighlight ti (explicit := false)
```

Comment:
```suggestion
          -- Keep the top-level tag so that users can also see the explicit version
          -- of the omitted term.
          pure <| some <| ← ppExprTagged ti.expr (explicit := false)
```
I think you want to keep the top-level tag here. Consider
```lean
set_option pp.deepTerms false in
set_option pp.maxTermDepth 2 in
#check (@id Nat 10).succ.succ.succ
```
In this term `@id Nat 10` is elided, and hovering over `⋯` shows `id 10` with the tag stripped. Then there is no way to see the explicit version `@id Nat 10`.

Reply:
Good catch!